device-registry/src/main/scala/org/genivi/sota/device_registry/DeviceRepository.scala

```diff
@@ -54,8 +56,28 @@ object Devices {
 
     def pk = primaryKey("id", id)
   }
+
+  type SystemInfoType = Json
+  case class SystemInfo(id: Device.Id, systemInfo: SystemInfoType)
+
+  class SystemInfoTable(tag: Tag) extends Table[SystemInfo] (tag, "DeviceSystem") {
+    def id = column[Id]("uuid")
+    def systemInfo = column[String]("system_info")
```

Comment:
this should be defined `column[Json]` and then you need to to define an implicit conversion from Json <-> String so that it can be written/read from the database. see http://slick.lightbend.com/doc/3.0.0/userdefined.html#using-custom-scalar-types-in-queries

Using these implicits instead of a this `convert` function will allow us to use all slick combinators to work with this table (filter, map, etc)